docs/kubernetes-installers.md

```diff
@@ -39,9 +39,18 @@ want. Once the cluster has been created, this CIDR cannot be changed.
 
 ### Rancher
 
-When creating a workload cluster, set the [network
-plugin](https://rancher.com/docs/rke/latest/en/config-options/add-ons/network-plugins/)
-to `none`.
+Follow these steps to deploy Antrea on [Rancher](https://ranchermanager.docs.rancher.com/pages-for-subheaders/rancher-on-a-single-node-with-docker) cluster via UI.
```

Comment:
It's a bit strange IMO that we are pointing to this document. The Rancher documentation has this as a secondary installation method, and clearly states:

> A Docker installation of Rancher is recommended only for development and testing purposes.

That's not really helpful for someone who wants to run Rancher + Antrea in production.

Reply:
@jainpulkit22 
There are different ways to install and use rancher in production/development environment , we might want to mention in above steps that these steps are only for development and testing purpose with antrea CNI , it is better to point to link https://ranchermanager.docs.rancher.com/v2.6/pages-for-subheaders/installation-and-upgrade wherein all types of installation steps are covered and also for custom cni installation option there is an another link https://rke.docs.rancher.com/config-options/add-ons/network-plugins/custom-network-plugin-example